.github/workflows/juju_3_6_ci.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Juju 3.6 CI tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    uses: canonical/data-platform-workflows/.github/workflows/lint.yaml@v18.0.0
+
+  build:
+    name: Build charm
+    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v18.0.0
+    with:
+      cache: true
+
+  integration-test-3-6:
+    strategy:
+      fail-fast: false
+      matrix:
+        architecture:
+          - amd64
+          - arm64
+    name: Integration | 3.6 | ${{ matrix.architecture }}
+    needs:
+      - lint
+      - build
+    uses: canonical/data-platform-workflows/.github/workflows/integration_test_charm.yaml@v18.0.0
+    with:
+      artifact-prefix: ${{ needs.build.outputs.artifact-prefix }}
+      architecture: ${{ matrix.architecture }}
+      cloud: microk8s
+      juju-snap-channel: 3.6/beta
+      libjuju-version-constraint: 3.5.2.0
+      microk8s-snap-channel: 1.28-strict/stable
```

Comment:
What about to add renovate label as discussed in MM?

Reply:
added renovate label in [5d740e3](https://github.com/canonical/mysql-k8s-operator/pull/490/commits/5d740e3252c1e9de6b2da8e5b40ef7c55c112748)